src/Calculator.ts

```diff
@@ -213,7 +211,63 @@ export function calculateMeasureReports(
         });
       }
 
-      createMeasureScore(measure, group, detail, report);
+      // if the measure definition has stratification, add group.stratifier
+      const stratifier = measure.group?.find(g => g.id == detail.groupId)?.stratifier;
+      if (stratifier) {
+        group.stratifier = [];
+        stratifier.forEach(s => {
```

Comment:
From the [structure def for MeasureReport](https://www.hl7.org/fhir/measurereport.html#invs):

> Stratifiers SHALL be either a single criteria or a set of criteria components (group.stratifier.stratum.all(value.exists() xor component.exists()))

This is easy to miss since it's an additional constraint that isn't in the structure definition. Since the TS interfaces are also based on the structure def, you won't get a TypeError either. I noticed this by trying to use the FHIR validator on the MeasureReport to ensure that the structure of the stratifier is in line with the spec

Reply:
So we need to add a value to these, I'm not entirely sure what this should be. It can probably just match the "text" that you put in the code property for now?